scalameta/semanticdb-scalac/src/main/scala/scala/meta/internal/semanticdb/DatabaseOps.scala

```diff
@@ -4,7 +4,7 @@ package semanticdb
 import scala.tools.nsc.Global
 
 trait DatabaseOps
-    extends AttributesOps
+    extends DocumentOps
```

Comment:
Can we keep alphabetic order here?

Reply:
Fixed, including other traits that were out of alphabetical order.